meson.options

```diff
@@ -3,7 +3,7 @@ option('bpf_clang', type: 'string', value: 'clang',
 option('bpftool', type: 'string', value: 'bpftool',
        description: 'bpftool to use when generating .bpf.skel.h')
 option('libbpf_a', type: 'string',
-       description: 'Static libbpf.a to use')
+       description: 'Static libbpf.a to use. Default is to use the one inside the libbpf submodule. Set this option to "disabled" to link libbpf dynamically.')
```

Comment:
Maybe sth like "By default, libbpf is automatically downloaded and built together. To use an existing static library, point this to the libbpf.a file or set it to "disabled" to use dynamic linking" would be clearer?